python/lsst/daf/base/exampleLib.py

```diff
@@ -0,0 +1,12 @@
+from _exampleLib import *
```

Comment:
In the case that @pschella mentioned, I think the only modules where `from x import *` would be used would the ones where C++ classes are further specialized (like the pure-Python modules Swig generates now).  I think of these as functionally similar to `__init__` modules (neither should appear in the fully-qualified `__module__` of the classes they contain, for instance).  Those files would indeed be harder to lint, but no others would be affected.